test/jdk/java/nio/channels/Selector/WakeupNow.java

```diff
@@ -47,33 +49,33 @@ private static void test1() throws Exception {
         // ensure wakeup is consumed by selectNow
         Thread.sleep(2000);
         sel.selectNow();
-        long startTime = System.currentTimeMillis();
+        long startTime = System.nanoTime();
         int n = sel.select(2000);
-        long endTime = System.currentTimeMillis();
+        long endTime = System.nanoTime();
         p.source().close();
         p.sink().close();
         sel.close();
-        if (endTime - startTime < 1000)
-            throw new RuntimeException("test failed");
+        long delta = endTime - startTime;
+        if (delta < 1_000_000_000)
+            throw new RuntimeException("test failed with delta " + delta);
     }
 
     // Test if selectNow clears wakeup with only the wakeup fd
     // in the selector.
     // This fails before the fix on Solaris
     private static void test2() throws Exception {
         Selector sel = Selector.open();
-        Pipe p = Pipe.open();
-        p.source().configureBlocking(false);
```

Comment:
Hello Brian, is this intentionally being removed?

Reply:
Looking at this test, this removal appears intentional and it's merely cleaning up unused (and thus irrelevant) `Pipe`. It appears that this was a copy/paste oversight in the original test (even the comment on `test2()` method seems to indicate this `Pipe` was an oversight). So what you did here looks correct.

Reply:
Hi @jaikiran . Yes it was unused code intentionally removed.